swagger.yaml

```diff
@@ -1681,6 +1843,62 @@ components:
         - START_TO_START
         - FINISH_TO_FINISH
 
+    Notification:
```

Comment:
Maybe add commentId also, so we can highlight the corresponding comment for the user, so we could give a direct link to the comment in the notifications list.

Reply:
In the next PR that introduces the comments :+1: